File: Tema3/Ejercicio1/ejercicio1.java
package Ejercicio1;

import java.util.Scanner;

class switches {
    public static void main(String[] args) {
        Scanner lector = new Scanner(System.in);
        System.out.println("Introduce un numero del 0 al 10 para asignar una nota: ");
        int notaUsuario = lector.nextInt();

        switch (notaUsuario) { // METER LOS CASE JUNTOS
            case 0:
                System.out.println("INSUFICIENTE");
                break;
            case 1:
                System.out.println("INSUFICIENTE");
                break;
            case 2:
                System.out.println("INSUFICIENTE");
                break;
            case 3:
                System.out.println("INSUFICIENTE");
                break;
            case 4:
                System.out.println("INSUFICIENTE");
                break;
            case 5:
                System.out.println("SUFICIENTE");
                break;
            case 6:
                System.out.println("BIEN");
                break;
            case 7:
                System.out.println("NOTABLE");
                break;
            case 8:
                System.out.println("NOTABLE");
                break;
            case 9:
                System.out.println("SOBRESALIENTE");
                break;
            case 10:
                System.out.println("SOBRESALIENTE");
                break;
            default:
                System.out.println("Nota inválida, asegurese de que la nota está entre 0 y 10.");
                break;
        }
        lector.close();
    }
}

File: Tema3/Ejercicio1/Ejercicio1.py
#coding: latin1

# Hacer una aplicación que con un switch se muestre por pantalla un mensaje si el 
# usuario ha introducido una nota, 1-4 INSUFICIENTE, 5 SUFICIENTE, 6 BIEN, 7-8 NOTABLE, 9 y 10 
#SOBRESALIENTE

def asignarNotas():
    nota = input("Introduce una nota, y el programa te dira a que rango corresponde")

    if nota > 0 and nota <= 10:
        if nota > 0 and nota < 5:
            print("INSUFICIENTE")
        elif nota == 5:
            print("SUFICIENTE")
        elif nota == 6:
            print("BIEN")
        elif nota > 6 and nota < 9:
            print("NOTABLE")
        elif nota >= 9 and nota <= 10:
            print("SOBRESALIENTE")
    else:
        print("Introduce una nota valida, del 1 al 10")

if __name__ == "__main__":
    asignarNotas()

# EN PYTHON NO HAY SWITCH POR LO QUE TUVE QUE HACERLO CON IF